bin/ck8s

```diff
@@ -10,37 +10,32 @@ source "${here}/common.bash"
 
 usage() {
     echo "COMMANDS:" 1>&2
-    echo "  init <wc|sc|both> [--generate-new-secrets]     initialize the config path" 1>&2
-    echo "  apply <wc|sc> [--sync] [--concurrency=<num>]   deploy the apps" 1>&2
-    echo "  test <wc|sc> [--logging-enabled]               test the applications" 1>&2
-    echo "  dry-run <wc|sc> [--kubectl]                    runs helmfile diff" 1>&2
-    echo "  fix-psp-violations <wc|sc>                     Checks and restarts pods that violates Pod Security Polices, applicable for new environments" 1>&2
-    echo "  upgrade <wc|sc|both> <vX.Y> prepare            runs all prepare steps upgrading the configuration" 1>&2
-    echo "  upgrade <wc|sc|both> <vX.Y> apply              runs all apply steps upgrading the environment" 1>&2
-    echo "  team add-pgp <fp>                              add a new PGP key to secrets" 1>&2
-    echo "  team remove-pgp <fp>                           remove a PGP key from secrets and rotate the data encryption key" 1>&2
-    # TODO: We might want to make this command less visible once we have proper
-    #       support for OIDC logins.
-    echo "  ops kubectl <wc|sc>                            run kubectl as cluster admin" 1>&2
-    echo "  ops kubecolor <wc|sc>                          run kubecolor as cluster admin" 1>&2
-    echo "  ops helm <wc|sc>                               run helm as cluster admin" 1>&2
-    # TODO: We might want to make this command less visible once we feel
```

Comment:
Do we want to remove such reminders?

Reply:
I am unsure, but the comments were 4 year old and personally I don't find them relevant as the `ops` commands are generally super useful as a Platform Administrator, hence I feel its good to keep them listed in the usage. It might however be a good idea to improve the usage, separate and group commands together etc. but that would be a separate issue. I don't have to address the `ck8s` usage here at all, but it might be good to improve it in the future. I would like to hear what others think about changing/updating the `ck8s` usage

Reply:
A lot has changed in 4 years (I think the comments might actually be even older than that). We are now moving towards making use of existing tooling directly rather than scripting around them. This means that hopefully commands like `ck8s apply` and `ck8s ops helm` will soon just become `helmfile apply` and `helm`.

TL;DR - the comments can be removed. :)